src/hidimstat/conditional_permutation_importance.py

```diff
@@ -75,6 +75,9 @@ def __init__(
             "categorical": imputation_model_categorical,
         }
         self.categorical_max_cardinality = categorical_max_cardinality
+        self.imputation_model_categorical = imputation_model_categorical
```

Comment:
The imputation models are already saved in a dictionary (lines 73-76):
```
 self.imputation_model = {
            "continuous": imputation_model_continuous,
            "categorical": imputation_model_categorical,
        }
```
I don't see the point to register them another time because any modification of one variable won't modify the second one. 
I think the best is to remove the lines 73 to 76 and to modify the rest of the code.

Reply:
For the above reasons, we need to have explicitly named attributes (e.g., self.imputation_model_continuous). So, I replaced the `imputation_model ` dict.